library/src/main/java/com/getstream/sdk/chat/utils/MessageListItemLiveData.java

```diff
@@ -135,60 +147,17 @@ public void observe(@NonNull LifecycleOwner owner,
             Log.i(TAG, "broadcast because reads changed");
             broadcastValue();
         });
```

Comment:
why are we making this change to this.messages.observe? @tbarbugli and @adrian09h

Reply:
For handle messageEntities of Thread Messages or Main Messages.